src/Esri.ArcGISRuntime.Toolkit/iOS/Esri.ArcGISRuntime.Toolkit.iOS/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Esri.ArcGISRuntime" version="100.1.0" targetFramework="xamarinios10" />
+  <package id="Esri.ArcGISRuntime.Xamarin.iOS" version="100.1.0" targetFramework="xamarinios10" />
```

Comment:
Surprisingly, no.  When you install the Esri.ArcGISRuntime package, the Esri.ArcGISRuntime.Xamarin.iOS reference also gets added.  I tested building with that removed, and it fails.